### PR TITLE
First dev commit

### PR DESCRIPTION
**Pull request by TychinaV**
scr/Platon.php - class was created to form a request for processing. src/Ordinary.php - class for the formation of a simple (form) request in processing.

### CODE REVIEW

File: src/Method/Ordinary.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Platon\Method;
+
+use InvalidArgumentException;
+
+
+class Ordinary extends Method
+{
+
```
Comment (aksafan) [inline src/Method/Ordinary.php:10]: Лишняя пустая строка, ну ты понял

File: src/Method/Ordinary.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Platon\Method;
+
+use InvalidArgumentException;
+
+
```
Comment (aksafan) [inline src/Method/Ordinary.php:7]: Лишняя пустая строка

File: src/Method/Ordinary.php (hunk 1 above)
Comment (TychinaV) [reply src/Method/Ordinary.php:10]: Bringing the coding style to the psr standard [pull-request](https://github.com/TychinaV/platon/pull/url)

File: src/Method/Ordinary.php (hunk 2 above)
Comment (TychinaV) [reply src/Method/Ordinary.php:7]: Bringing the coding style to the psr standard [pull-request](https://github.com/TychinaV/platon/pull/url)

**Pull request merged**